packages/cmrjs/src/cmr-utils.js

```diff
@@ -64,7 +64,11 @@ function getFileDescription(file, urlType = 'distribution') {
 
 const isECHO10File = (filename) => filename.endsWith('cmr.xml');
 const isUMMGFile = (filename) => filename.endsWith('cmr.json');
-const isCMRFilename = (filename) => isECHO10File(filename) || isUMMGFile(filename);
+const isISOFile = (filename) => filename.endsWith('cmr_iso.xml');
+// eslint-disable-next-line max-len
```

Comment:
Do you still need this?

Reply:
definitely didn't mean to leave that in there~ thanks for letting me know, it's been removed [`b9fe488` (#2226)](https://github.com/nasa/cumulus/pull/2226/commits/b9fe4886adc6c8e00996252e9e65e9d2529134c5)